content/ember-5-0-released.md

```diff
@@ -0,0 +1,233 @@
+    We shipped support for `<template>` tags in `.gjs` and `.gts` files over the course of 2022.
+    That includes [ESLint integration](https://github.com/ember-cli/eslint-plugin-ember) and [a Prettier plugin](https://github.com/gitKrystan/prettier-plugin-ember-template-tag), as well as [ember-template-lint](https://github.com/ember-template-lint/ember-template-lint) support!
+    We expect to recommend `<template>` as the default authoring format during Ember 5.x.
+
+- We [shipped Glint 1.0](https://blog.emberjs.com/glint-1-0-released/), providing top-notch editor tooling for Ember's Handlebars templates in both JavaScript and TypeScript code bases, powered by TypeScript under the hood.
+
+- Ember Data [massively improved performance](https://blog.emberjs.com/ember-released-4-7/) and [laid a foundation](https://blog.emberjs.com/ember-data-5-x-update-2023-04-15/) for a total overhaul in the 5.x series.
+
+Ember 5.0 has no new features, because we did all that work in minor releases instead.
+
+You can read more about our general release process here:
+
+- [Release Dashboard](http://emberjs.com/releases/)
+- [The Ember Release Cycle](https://blog.emberjs.com/new-ember-release-process/)
+- [The Ember Project](https://blog.emberjs.com/ember-project-at-2-0/)
+- [Ember LTS Releases](https://blog.emberjs.com/announcing-embers-first-lts/)
+
+## Beta cycle
+
+This release kicks off the 5.1 beta cycle for all sub-projects. We encourage our community (especially addon authors) to help test these beta builds and report any bugs before they are published as a final release in six weeks' time. The [ember-try](https://github.com/ember-cli/ember-try) addon is a great way to continuously test your projects against the latest Ember releases.
+
+## How to upgrade to Ember 5
+
+The most common approach for upgrading to 5.0 is to upgrade your app to the last
+version of Ember 4, which is 4.12, resolve all deprecation warnings, and then upgrade to 5.0. If your app or addon runs with no deprecations in the latest release of 4.12,
+you should be able to upgrade to Ember 5 with no additional changes. Step by step directions are below.
+
+If your app is at a much earlier version in the 4.x series, we
+recommend upgrading in steps across the LTS versions: 4.4, 4.8, and then 4.12.
+
+Follow these steps in order:
+
+1. Upgrade your project to the latest patch version of Ember 4.12.
+   Many developers can do this by running
+   `npx ember-cli-update --to 4.12`. The
+   [general Ember upgrade guide](https://cli.emberjs.com/release/basic-use/upgrading/#upgradinganemberappitself)
+   has more details about how to upgrade your Ember app/addon.
+2. Make sure your app builds successfully.
+3. Resolve all deprecation warnings. These Deprecated APIs are removed in Ember 5.0. You may need to upgrade some of your addon dependencies if they are using deprecated APIs. 
```

Comment:
Not sure if I should mention the `@ember/string` deprecation bug here too, since it may be impossible to clear that deprecation.

Reply:
I think so. That's a good call-out. I can add a quick note. I'm working on some of the other content tonight.